src/main/java/com/marklogic/client/ext/document/ContentIdExtractor.java

```diff
@@ -1,9 +1,13 @@
 package com.marklogic.client.ext.document;
 
+import java.io.IOException;
+
 import com.marklogic.client.io.marker.AbstractWriteHandle;
 
+//@FunctionalInterface
```

Comment:
Is this needed? If not, I recommend removing this line - commented-out code can cause confusion.